openlane/scripts/openroad/gpl.tcl

```diff
@@ -53,8 +53,7 @@ if { $::env(PL_SKIP_INITIAL_PLACEMENT) } {
 	lappend arg_list -skip_initial_place
 }
 
-
-if { [info exists ::env(__PL_SKIP_IO)] } {
+if { $::env(STEP_ID) == "OpenROAD.GlobalPlacementSkipIO" } {
```

Comment:
i recall avoiding this for some reason.

Reply:
right, because inheriting steps would break